drivers/src/trng.rs

```diff
@@ -66,3 +76,44 @@ impl Trng {
         }
     }
 }
+
+pub struct FakeRng {}
```

Comment:
I don't think this struct is necessary; just return the bogus data in the Trng::generate() match arm.